xrandr_test.go

```diff
@@ -1,9 +1,10 @@
 package xrandr_test
 
 import (
+	"testing"
```

Comment:
Sorry, looks like gofmt/goimports did some drive-by changes here. Should I revert them?

Reply:
it's ok.